compiler/src/dmd/iasmgcc.d

```diff
@@ -384,6 +385,9 @@ unittest
 {
     import dmd.mtype : TypeBasic;
 
+    if (!global.errorSink)
+        global.errorSink = new ErrorSinkCompiler;
```

Comment:
Likewise, this pattern is not good

Reply:
You're right, that's left over from a previous incarnation of this PR. I had a fair amount of trouble figuring out where the unittests were being initialized.

Reply:
Unfortunately, this code doesn't call globals._init, and so global.errorSink is not initialized. So I have to initialize it here.

Reply:
Having everything in global is not ideal either.